clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java

```diff
@@ -285,24 +310,24 @@ private EhcacheEntityResponse invokeLifeCycleOperation(ClientDescriptor clientDe
       case DESTROY_SERVER_STORE:
         destroyServerStore(clientDescriptor, (DestroyServerStore) message);
         break;
+      case CREATE_CLUSTERED_MAP:
+        createConcurrentMap(clientDescriptor, (CreateClusteredMap) message);
```

Comment:
I don't know that seems like a bit of a premature optimization... it's only 1 extra message right?